docs/src/app/components/pages/components/icon-menus.jsx

```diff
@@ -71,6 +71,12 @@ export default class IconMenus extends React.Component {
             desc: 'If true, menu will close after an item is touchTapped.',
           },
           {
+            name: 'open',
+            type: 'bool',
+            header: 'default: true',
```

Comment:
If you mean this is optional prop, `undefined` is more sense. `null` mean, the user have provide a value, and the value is `null`.

For example, in CoffeeScript
```
foo?.bar
```

turn into 

```
if (typeof foo !== "undefined" && foo !== null) {
  foo.bar;
}
```